daphne_worker/src/lib.rs

```diff
@@ -492,6 +492,13 @@ impl DaphneWorkerRouter {
             )])
             .inc();
 
+        // Push metrics to Prometheus metrics server, if configured.
```

Comment:
It would be cool if we could have some sort of "do this before you shut me down, or after n seconds" thing, as then we could amortize the cost of stats uploads across multiple invocations in the same isolate, much like we're caching config.  I haven't seen any API to do that though.